app/views/admin/videos/index.html.erb

```diff
@@ -13,3 +13,28 @@
   <br>
   <%= f.submit "Submit", class: "btn btn-primary col-sm-2 col-sm-offset-5" %>
 <% end %>
+
+<div class="row table-index">
+  <h3 class="text-center"><u>All Videos</u></h3>
+  <table class="table table-condensed">
+    <thead>
+    <tr class="warning">
+      <th data-field='title'>Title</th>
+      <th data-field='url'>Url</th>
+      <th data-field='updated_at'>Last Updated</th>
+      <th data-field="Edit/Delete">Edit/Delete</th>
+    </tr>
+    </thead>
+
+    <tbody>
+    <% @videos.each do |video| %>
+      <tr id="table-row" data-title= "<%= video.title %>">
+        <td><%= video.title %></td>
+        <td><%= video.url %></td>
+        <td> <%= link_to "Edit", edit_admin_video_path(video), class: "btn btn-primary" %>
```

Comment:
I guess same question here.  Are we still planning to make a CMS page for videos and pdfs?  If so, do we need the ability to edit a video?  This would allow people to save a step if they just want to throw a new title & url in there.  Would still have the same slot and therefore be automatically placed on the page, negating the necessity of using our CMS.